### ISSUE TITLE
Random Resprings

### ISSUE THREAD

**Issue by FeverEclipse** [coolstar/electra1131#2]
-According to CrashReporter- Springboard crashes even when NO TWEAKS INSTALLED(Except the ones came with electra itself).Using an A9(6s) device.So i think Substrate/Substitute/Tweak Injector causes this.And i’m not the only one(i’ve met with 4 people that have this problem)

Edit:You can find crash log here https://drive.google.com/open?id=19ZMwy0l4LHBeDiesbJGsF_CGhmIEYu57

**Comment by airsquared** [comment]
It'll be more useful to Coolstar if you shared the log

**Comment by akasarx** [comment]
update everything in cydia and make sure to have battery charged as mine resprings when it's low

**Comment by maikoross** [comment]
Hi all, so I finally found out why my phone kept respringing randomly into safemode despite having everything removed. If you have removed all tweaks and still get alot of random resprings into safe mode, follow these steps.

1.Install iCleaner Pro from this repo: http:cydia.angelxwind.net/

2.Click the Plus sign in the bottom left hand corner

3.Tap on Cydia Substrate Addons

4.Scroll right down until you see XMHelper.dylib

5.Disable this addon and respring your device

I disabled this addon yesterday and have not had a single respring into safemode since. I have re-installed the tweaks listed below and my phone has been stable.
